main.tf

```diff
@@ -1,32 +1,44 @@
+locals {
+  values = yamlencode({
+    "awsRegion" : data.aws_region.current.name,
+    "autoDiscovery" : {
+      "clusterName" : var.cluster_name
+    },
+    "rbac" : {
+      "create" : true,
+      "serviceAccount" : {
+        "create" : true,
```

Comment:
maybe we want to pass this through a variable?

Reply:
make sense, done